client/store/orders.js

```diff
@@ -0,0 +1,71 @@
+import axios from 'axios'
+import history from '../history'
+import {addItems} from './orderItems'
+/**
+ * ACTION TYPES
+ */
+const GET_ORDERS = 'GET_ORDERS'
+const ADD_ORDER = 'ADD_ORDER'
+/**
+ * INITIAL STATE
+ */
+
+/**
```

Comment:
example redux store:
shoppingHistory:[[orderItems from some order], []],
currentCart:[...orderItems]

original data:
{orderItemId: 1,
product:{
name:''
}
}

=> {productName: '', quantity: 0}
look into the {raw:true} option in sequelize